forwarder_launch.py

```diff
@@ -84,13 +84,15 @@
 
     statistic_reporter = None
     if grafana_carbon_address:
+        statistics_update_interval = args.statistics_update_interval
```

Comment:
no need to put this in a variable. Just construct the object with args._statistics_update_interval.